PmipCallStackFilter.cs

```diff
@@ -50,70 +56,78 @@ public static DkmStackWalkFrame PmipStackFrame(DkmStackContext stackContext, Dkm
             return frame;
         }
 
-        public static void RefreshStackData(string fileName)
+        public static void RefreshStackData(int pid)
         {
-            try
-            {
-                if (!File.Exists(fileName))
-                    return;
+            DirectoryInfo taskDirectory = new DirectoryInfo(Path.GetTempPath());
+            FileInfo[] taskFiles = taskDirectory.GetFiles("pmip_" + pid + "_*.txt");
+
+            if (taskFiles.Length < 1)
+                return;
+
+            Array.Sort(taskFiles, (a, b) => Path.GetFileNameWithoutExtension(a.Name).CompareTo(Path.GetFileNameWithoutExtension(b.Name)));
```

Comment:
Will this correctly compare Foo1.txt and Foo10.txt?

Reply:
Probably not. I'll instead parse the number out of the filename and compare those as integers.